airflow/utils/context.py

```diff
@@ -170,106 +149,6 @@ def _get_asset_from_db(name: str | None = None, uri: str | None = None) -> Asset
         return asset.to_public()
 
 
-class LazyAssetEventSelectSequence(LazySelectSequence[AssetEvent]):
-    """
-    List-like interface to lazily access AssetEvent rows.
-
-    :meta private:
-    """
-
-    @staticmethod
-    def _rebuild_select(stmt: TextClause) -> Select:
-        return select(AssetEvent).from_statement(stmt)
-
-    @staticmethod
-    def _process_row(row: Row) -> AssetEvent:
-        return row[0]
-
-
-@attrs.define(init=False)
-class InletEventsAccessors(Mapping[Union[int, Asset, AssetAlias, AssetRef], LazyAssetEventSelectSequence]):
-    """
-    Lazy mapping for inlet asset events accessors.
-
-    :meta private:
-    """
-
-    _inlets: list[Any]
-    _assets: dict[AssetUniqueKey, Asset]
-    _asset_aliases: dict[AssetAliasUniqueKey, AssetAlias]
-    _session: Session
```

Comment:
Would this be a breaking change?

Reply:
Nope, this one should be fine. But I did miss another breaking change. I'll update it

Reply:
just added `newsfragments/45960.significant.rst`